tests/storage/snapshots/conftest.py

```
@@ -149,3 +173,202 @@ def vm_restore_with_predictable_names(
+        name=f"restore-{vm_with_4_disks_for_snapshot.name}",
+        namespace=vm_with_4_disks_for_snapshot.namespace,
+        vm_name=vm_with_4_disks_for_snapshot.name,
+        snapshot_name=snapshot_of_vm_with_4_disks.name,
+        client=admin_client,
+    ) as vm_restore:
+        vm_restore.wait_restore_done(timeout=TIMEOUT_5MIN)
+        yield vm_restore
+
+
+@pytest.fixture()
+def restored_four_vms(admin_client, four_vms_with_4_disks_for_snapshot, snapshots_of_four_vms):
```

Ruff (0.16.1)

[warning] Unused function argument: `four_vms_with_4_disks_for_snapshot`

(ARG001)